plaid/client.py

```diff
@@ -102,6 +106,8 @@ def _post(self, path, data, is_json):
         headers = {}
         if self.api_version is not None:
             headers = {'Plaid-Version': self.api_version}
+        if self.app_name is not None:
+            headers['Client-App'] = self.app_name
```

Comment:
Can we name the header `Plaid-Client-App` to align with https://github.com/plaid/plaid-node/pull/196/files#diff-f1e95dd7e502ba1d181d7b7f83bd4cefR75